include/nbl/asset/filters/CBlitImageFilter.h

```diff
@@ -465,7 +530,7 @@ class CBlitImageFilter : public CImageFilter<CBlitImageFilter<Normalize,Clamp,Sw
 							lineBuffer = intermediateStorage[axis-1]+core::dot(static_cast<const core::vectorSIMDi32&>(intermediateStrides[axis-1]),localTexCoord)[0];
 						else
 						{
-							const auto windowEnd = inExtent.width+window_last.x;
+							const auto windowEnd = inExtent.width+window_last.x+1;
```

Comment:
you'll overwrite your own linebuffer

Reply:
i now realize its basically same as putting a `<=` in the `for` below (apart from other consequences of `window_last` ofcourse)

Reply:
I think this is already fixed in [this](https://github.com/Devsh-Graphics-Programming/Nabla/commit/d91717e6fe5b9a655bf851bcdca424dafe6e5ff0) commit. 

Resolved?

Reply:
yeah, you were right about the -1 being wrong.

a few renamings are necessary:
https://github.com/Devsh-Graphics-Programming/Nabla/pull/140#discussion_r863651029


Also I didn't get to comment on this line (because it never changed), but this:
```cpp
							const auto windowEnd = inExtent.width+window_end.x;
							decode_offset = alloc_decode_scratch();
							lineBuffer = intermediateStorage[1]+decode_offset*MaxChannels*windowEnd;
							for (auto& i=localTexCoord.x; i<windowEnd; i++)
```
should be renamed for clarity into this
```cpp
							const auto inputEnd = inExtent.width+real_window_size.x;
							decode_offset = alloc_decode_scratch();
							lineBuffer = intermediateStorage[1]+decode_offset*MaxChannels*inputEnd;
							for (auto& i=localTexCoord.x; i<inputEnd; i++)
```

Reply:
Resolved in #343.